concepts/observability-configurable-monitoring/05-concept.md

```diff
@@ -15,7 +15,7 @@ To assure that the scraping collector will scale with the targets to scrape, it
 
 **Push support will be provided via OTLP only via the central deployment**
```

Comment:
```suggestion
**Push support will be provided using OTLP only with the central deployment**
```
We [don't like](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#terminology) "via".